flink-table/flink-table-common/src/test/java/org/apache/flink/table/catalog/CatalogTestBase.java

```diff
@@ -716,4 +1091,8 @@ protected void checkEquals(CatalogView v1, CatalogView v2) {
 		assertEquals(v1.getOriginalQuery(), v2.getOriginalQuery());
 		assertEquals(v1.getExpandedQuery(), v2.getExpandedQuery());
 	}
+
+	protected void checkEquals(CatalogPartition expected, CatalogPartition actual) {
+		assertEquals(expected.getProperties(), actual.getProperties());
```

Comment:
also check partition comment?

Reply:
CatalogPartition doesn't have comment. I'll check descriptions instead.

Reply:
oh, then probably don't need to do so, we don't check descriptions so far yet